ui/lib/pki/addon/templates/keys.hbs

```diff
@@ -7,4 +7,5 @@
     model=this.model.id
   }}
   @isEngine={{true}}
-/>
\ No newline at end of file
+/>
+{{outlet}}
```

Comment:
Same question here as the [Issuer List PR ](https://github.com/hashicorp/vault/pull/17236/files#r975816814)

Reply:
It's a great point. Will revisit after we get all the routes headers generated and the pattern becomes a little clearer.